libvcx/src/aries/handlers/issuance/holder/holder.rs

```diff
@@ -84,14 +92,14 @@ impl Holder {
 
     pub fn update_state(&mut self, connection: &Connection) -> VcxResult<u32> {
```

Comment:
should return `VcxResult<HolderState>`

Reply:
That's true, good catch, don't know how I missed that